### PR TITLE
Msk

### PR DESCRIPTION
**Pull request by awssteph**
Added MSK Query

### CODE REVIEW

File: static/Cost/300_Optimization_Data_Collection/Code/Optimization_Data_Collector.yaml [hunk 1]
```diff
@@ -390,7 +390,7 @@ Resources:
           GlueRoleArn: !GetAtt GlueRole.Arn 
           MultiAccountRoleName: !Sub "${RolePrefix}${MultiAccountRoleName}"
           RolePrefix: !Ref RolePrefix
-      TemplateURL: "https://aws-well-architected-labs.s3-us-west-2.amazonaws.com/Cost/Labs/300_Optimization_Data_Collection/budgets.yaml"
```
Comment (iakov-aws) [inline static/Cost/300_Optimization_Data_Collection/Code/Optimization_Data_Collector.yaml:393]: budgets_dev?

File: content/Cost/300_Labs/300_CUR_Queries/Queries/analytics.md [hunk 2]
```diff
@@ -359,6 +360,49 @@ ORDER BY
 
 [Back to Table of Contents](#table-of-contents)
 
+
+
+
+### Amazon MSK
+
+#### Query Description
```
Comment (amazonchris) [inline content/Cost/300_Labs/300_CUR_Queries/Queries/analytics.md:368]: Should we call out in the description that the query would cover serverless ?

**Review by amazonchris — COMMENTED** [review]
- Tested query with known dataset and the result matches against CE down to the penny
- For the image, the accounts shown are not the standard 111222333444 etc... I would recommend doing away with the image showing the query result, we've been moving away from this.

File: static/Cost/300_Optimization_Data_Collection/Code/Optimization_Data_Collector.yaml (hunk 1 above)
Comment (awssteph) [reply static/Cost/300_Optimization_Data_Collection/Code/Optimization_Data_Collector.yaml]: good spot, two things at once

File: content/Cost/300_Labs/300_CUR_Queries/Queries/analytics.md (hunk 2 above)
Comment (awssteph) [reply content/Cost/300_Labs/300_CUR_Queries/Queries/analytics.md:368]: done

**Review by amazonchris — APPROVED** [review]
Tested with submitted changes.

**Pull request merged**